### PR TITLE
Feat : Text Utility Component

### PR DESCRIPTION
**Pull request by anuj-kumary**

### CODE REVIEW

File: components/Text-Utility/text.css [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    color: var(--text-color);
+}
+
+.paragraph {
+    padding: 0.5rem;
+    margin: 0.5rem;
+}
+
+.horizonatl-line {
+    margin: 2rem 0;
```
Comment (thegeorgenikhil) [inline components/Text-Utility/text.css:72]: typo

**Review by vinc3nati — APPROVED** [review]
Looks good.

**Review by thegeorgenikhil — CHANGES_REQUESTED** [review]
Approving this

**Review by thegeorgenikhil — APPROVED** [review]

**Review by hrishi82 — APPROVED** [review]
Looks good to merge

**Review by varnita619 — APPROVED** [review]
looks good to go

**Pull request merged**